apps/client/src/features/event-editor/EventEditorExport.tsx

```diff
@@ -19,11 +19,13 @@ const closeBtnStyle = {
 };
 
 const EventEditorExport = () => {
-  const editId = useAppMode((state) => state.editId);
-  const setEditId = useAppMode((state) => state.setEditId);
+  const { clearEventsToEdit, eventsToEdit } = useAppMode();
 
-  const editorStyle = cx([style.eventEditor, !editId ? style.noEvent : null]);
-  const removeOpenEvent = () => setEditId(null);
+  const editorStyle = cx([
+    style.eventEditor,
+    eventsToEdit.length > 1 || eventsToEdit.length === 0 ? style.noEvent : null,
```

Comment:
the code in question is likely not here, using this as a discussion place.

With the current changes we lost the distinction between play and edit mode.

The EventEditor (here) should only open for selected events in edit mode